src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -530,6 +534,24 @@ function processItem(item, transaction) {
 }
 ```
 
+#### Retrieving a Transaction
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction otherwise it returns `undefined`. If you are using our Tracing integration by default we attach the Transaction to the Scope. So you could do something like this:
```

Comment:
```suggestion
In cases where you want to attach a span to already ongoing transaction, you can use `Sentry.getCurrentHub().getScope().getTransaction()` call. This function will return a `Transaction` in case there is a running transaction or `undefined` otherwise. If you are using our Tracing integration, we attach the transaction to the scope by default, so you could do something like this:
```